exec/java-exec/src/main/java/org/apache/drill/exec/store/pcap/PcapBatchReader.java

```diff
@@ -406,8 +406,8 @@ private void addDataToTable(Packet packet, int networkType, RowSetLoader rowWrit
     srcMacAddressWriter.setString(packet.getEthernetSource());
     dstMacAddressWriter.setString(packet.getEthernetDestination());
 
-    dstIPWriter.setString(packet.getDst_ip().getHostAddress());
-    srcIPWriter.setString(packet.getSrc_ip().getHostAddress());
+    dstIPWriter.setString(packet.getDst_ip_String());
```

Comment:
Is it possible to call `setNull()` for null values instead of propagating empty strings?

Reply:
I was trying to keep the null check out of the batch reader, but in the end I had to put it there.  So yes... now this is addressed and the values are null if in fact that's the case.